sockeye/training.py

```diff
@@ -692,14 +702,16 @@ def _update_metrics(self,
             if result is not None:
                 decoded_checkpoint, decoder_metrics = result
                 self.state.metrics[decoded_checkpoint - 1].update(decoder_metrics)
-                if self.tensorboard_logger is not None:
-                    self.tensorboard_logger.log_metrics(decoder_metrics, decoded_checkpoint)
+                self.tflogger.log_metrics(decoder_metrics, decoded_checkpoint)
             process_manager.start_decoder(self.state.checkpoint)
 
         self.state.metrics.append(checkpoint_metrics)
         utils.write_metrics_file(self.state.metrics, self.metrics_fname)
-        if self.tensorboard_logger is not None:
-            self.tensorboard_logger.log_metrics(checkpoint_metrics, self.state.checkpoint)
+
+        tf_metrics = checkpoint_metrics.copy()
+        tf_metrics.update({"%s_grad" % n: v for n, v in self.state.gradients.items()})
+        tf_metrics.update(self.model.module.get_params()[0])
```

Comment:
for readability it might make sense to manually destruct to arg_params and aux_params

Reply:
done!